src/Editor/Util/AssetMetaData.cpp

```diff
@@ -1,37 +1,65 @@
 #include "AssetMetaData.hpp"
+#include <fstream>
+#include <Utility/Log.hpp>
 
-bool AssetMetaData::GenerateMetaData(const char * filepath, MeshImportData * meshImportData) {
-    // Open file for writing.
+bool AssetMetaData::SaveMetaData(const char* filepath, MeshMetaData* metaData) {
+    Json::Value node;
+    ToJson(metaData, node);
+    Save(filepath, node);
+    return true;
```

Comment:
Always returns true. Either change the the return value so it doesn't claim to return whether it succeeded when it doesn't, or change the function so it actually returns whether it succeeded.

`Save` returns whether the file could be opened, so just return whatever `Save` returns.